extensions/amp-image-slider/0.1/amp-image-slider.js

```diff
@@ -42,10 +45,11 @@ export class AmpImageSlider extends AMP.BaseElement {
     this.container_ = null;
 
     /** @private {?Element} */
-    this.leftAmpImage_ = null;
-
+    this.leftImage_ = null;
     /** @private {?Element} */
-    this.rightAmpImage_ = null;
+    this.rightImage_ = null;
+    /** @private {boolean} */
```

Comment:
Nit: whitespace convention
```suggestion

    /** @private {boolean} */
```

Reply:
This convention is not equally followed within the file, going to leave this one be.